StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -50,11 +46,20 @@ import UIKit
                     return nil
                 }
             }()
-            stackView.addArrangedSubview(RowButton.makeForSavedPaymentMethod(paymentMethod: savedPaymentMethod,
-                                                                             appearance: rowButtonAppearance,
-                                                                             rightAccessoryView: accessoryButton,
-                                                                             isEmbedded: true,
-                                                                             didTap: handleRowSelection(selectedRowButton:)))
```

Comment:
Ah this makes me wonder, do the other RowButtons have a retain cycle if they pass a selector directly to didTap?   Like on line 44